src/tests/ftest/server/metadata.py

```diff
@@ -357,3 +357,73 @@ def test_metadata_server_restart(self):
 
                 # Start the servers
                 self.start_server_managers()
+
+    @avocado.fail_on(DaosApiError)
+    def test_container_removal_after_der_nospace(self):
+        """JIRA ID: DAOS-4858.
+
+        Test Description:
+           Verify container can be successfully deleted when the storage pool
+           is full ACL grant/remove modification.
```

Comment:
```suggestion
           is full.
```
I don't see any ACL related step. Am I missing something?

Reply:
Thanks for the comment.
This test is to create out of space error by using the default ACL for pool and containers.